### DISCUSSION TITLE
cert-manager kubernetes acme to internal smallstep CA

### DISCUSSION THREAD

**Discussion by reesericci** [smallstep/certificates#455] [Q&A] [answered]
Wondering how to make cert-manager trust smallstep so that it will be able to get certificates via acme

**Comment by maraino** [comment, marked as answer]
Hi @reesericci, 

I'm not sure if this is the right solution, but @tashian wrote a blog post using Google CAS, step-ca and cert-manager , there's a step where the cert-manager deployment is patched with a custom root certificate.  Here it is https://smallstep.com/blog/acme-ra-gcp-cas/

@tashian Is that the right approach?

**Reply by reesericci** [reply]
Also, I made an issue on autocert asking for ingress support. smallstep/autocert/issues/27

**Reply by tashian** [reply]
Yes, this is the right approach. cert-manager doesn't have an easy configuration option for adding items to the trust store, so you need to patch it using my instructions in the post: https://smallstep.com/blog/acme-ra-gcp-cas/
